test/index.spec.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA ;
+ */
+
+const postcss = require('postcss');
+
+const plugin = require('../index.js');
+
+function run(input) {
+    return postcss([
+        require('postcss-preset-env')({
+            stage: 1,
+            features: {
+                'nesting-rules': {
+                    noIsPseudoSelector: true
+                }
+            },
+            insertAfter: {
+                'nesting-rules': plugin
+            }
+        })
+    ]).process(input);
```

Comment:
chore: To avoid a console warning, you need to add the option `{ from: void 0 }`
```suggestion
    ]).process(input, { from: void 0 });
```